docs/src/main/paradox/getting-started.md

```diff
@@ -0,0 +1,107 @@
+It builds a full @ref:[CQRS](use-cases.md#command-query-responsibility-segregation-cqrs-) (Command Query Responsibility Segregation) ES (Event Sourcing) system using a combination of features from the akka toolkit.
+
+## Part 1: Setup your application
+
+Add the Akka Projections core library to a new project.
+This isn't strictly required, because as we add other dependencies in the following steps it will transitively include core as a dependency, but it never hurts to be explicit.
+
+@@dependency [sbt,Maven,Gradle] {
+  group=com.lightbend.akka
+  artifact=akka-projection-core_$scala.binary.version$
+  version=$project.version$
+}
+
+Define the event type protocol that will represent each `Envelope` streamed from the Source Provider.
+Create a typed @apidoc[akka.actor.typed.ActorSystem] for various Projections implementations to use.
+
+Scala
+:  @@snip [ShoppingCartApp.scala](/examples/src/test/scala/docs/guide/ShoppingCartApp.scala) { #guideSetup }
+
+## Part 1: Choosing a Source Provider
+
+A @apidoc:[SourceProvider] will provide the data to our projection. In Projections data is referenced as an `Envelope`, but it could reference an event, a message from a queueing system, or a record from a database table.
+There are several supported Source Provider's available (or you can build your own), but in this example we will use the @ref:[Event Sourced Source Provider](eventsourced.md).
+
+Add the following dependencies to your project:
+
+@@dependency [sbt,Maven,Gradle] {
+  group=com.lightbend.akka
+  artifact=akka-projection-eventsourced_$scala.binary.version$
+  version=$project.version$
+}
+
+Add the following imports to your project:
+
+Scala
+:  @@snip [ShoppingCartApp.scala](/examples/src/test/scala/docs/guide/ShoppingCartApp.scala) { #guideSourceProviderImports }
+
+Create the @apidoc[SourceProvider].
+The @ref:[Event Sourced Source Provider](eventsourced.md) is using [Akka Persistence](https://doc.akka.io/docs/akka/current/typed/persistence.html) internally (specifically the [eventsByTag](https://doc.akka.io/docs/akka/current/persistence-query.html#eventsbytag-and-currenteventsbytag) API).
+To initialize the Source Provider we need to set parameters to choose the Akka Persistence plugin (Cassandra) to use as well as the name of the tag used for events we're interested in from the journal.
+
+Scala
+:  @@snip [ShoppingCartApp.scala](/examples/src/test/scala/docs/guide/ShoppingCartApp.scala) { #guideSourceProviderSetup }
+
+## Part 2: Build a simple Projection handler
+
+While building a projection there are several non-functional requirements to consider.
+_What technology to project into? What message delivery semantics are acceptable for the system? Is it compatible with the chosen Source Provider to enable exactly-once message delivery? Does runtime state need to be maintained in the projection while it's running?_
```

Comment:
Love this list! I would even make it a list so it's some sort of checklist.
We could even add "Is the _source_ sharded (or partitioned)?".

In the following paragraph, then, you can list the answers to each item on the checklist.

Reply:
Maybe "Is the source sharded (or partitioned)?" is too advanced for a section this early in the GettingStarted guide.

Reply:
Thanks.  Maybe some of this could move to other parts of the docs where it would be more appropriate.  I'll consider adding answers to these questions, relevant to the example itself.